pymepixviewer/main.py

```diff
@@ -415,11 +424,19 @@ def onModeChange(self, value):
 
     def onData(self, data_type, event):
 
+        print(data_type)
```

Comment:
Do we want these prints in there? If we need an output here, this should rather be `logging`.

Reply:
The same, outdated.